commands/exceptions.py

```diff
@@ -0,0 +1,26 @@
+from commands.status import Status
+
+class FileNotFound(Exception):
+    def __init__(self, message):
+        self.status = Status.FileNotFound
+        self.message = message
```

Comment:
Those classes extends from Exception class.
The Exception class (parent class ) has a message is an instance variable.

you don’t need to duplicate this in the child class
What you need here is to use 
Super(message) in every __init.